src/main/common/maths.c

```diff
@@ -111,7 +111,7 @@ int32_t applyDeadband(int32_t value, int32_t deadband)
     return value;
 }
 
-inline int constrain(int amt, int low, int high)
```

Comment:
ok didnt realize that. Thanks.

I think constrain functions need to be inlined for sure it is used all the
time
Op 30 apr. 2016 17:15 schreef "John Aughey" <notifications@github.com>:

> In src/main/common/maths.c
> <https://github.com/borisbstyle/betaflight/pull/372#discussion_r61670856>:
>
> > @@ -111,7 +111,7 @@ int32_t applyDeadband(int32_t value, int32_t deadband)
> >      return value;
> >  }
> >
> > -inline int constrain(int amt, int low, int high)
>
> The source for constrain must be in a .h file for other .c files to use
> it. To inline, compilers must have the source for the function available.
> Inline is only a suggestion, so it's not actually being inlined, except for
> maybe functions inside of maths.c.
>
> If you want it to be inlined, we can move the code to maths.h.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/borisbstyle/betaflight/pull/372/files/daf1e239ebd8655a9ef32e5a8d98984926808c7c#r61670856>
>